lib/components/table.vue

```diff
@@ -127,12 +131,11 @@
                         return this.sortDesc ? r : r * -1;
                     });
                 }
-
+                this.$emit('input', items.length);
```

Comment:
We should keep tables more abstract. If only binds to length we for example can't receive new data. Why not simply binding model to `items` itself? Also please consider that this emit is in a computed property! we should somehow keep latest value and compare before this event to prevent duplicate ones :)